### PR TITLE
Check pipe version test

### PR DESCRIPTION
**Pull request by maryvictol**
This PR adds the test verifying update pipe version in the existing run.

### CODE REVIEW

File: docs/testcases/UI/Test_Environment/TC-TEST-ENVIRONMENT-3.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+| 7 | Click **Generate access key** button | |
+| 8 | Store *CLI configure command* | |
+| 9 | Open the **Runs** page | |
+| 10 | Click the run launched at step 3 of the Preparations | |
+| 11 | Click the **SSH** hyperlink | |
+| 12 | In the opened tab, enter and perform the command: `pipe --version` | |
+| 13 | Enter and perform the command: `pipe ssh <runID>`, where `<runID>` is ID of run launched at step 3 of the Preparations | Interactive session over the SSH protocol is started |
+| 14 | Enter and perform the commands from *Pipe CLI Installation Content* stored at step 6 | |
+| 15 | Enter and perform the *CLI configure command* stored at step 8 | |
+| 16 | Enter and perform the command: `<pipe_new_version>/pipe --version`, where `<pipe_new_version>` is the path of the new version pipe CLI installed at step 14 |  |
```
Comment (NShaforostov) [inline docs/testcases/UI/Test_Environment/TC-TEST-ENVIRONMENT-3.md]: Add expected result for the action: `Version differs from the version in output of step 13`

**Review by NShaforostov — APPROVED** [review]
Before merging, fix a minor note, please - see the comment.

File: e2e/gui/src/test/resources/prerequisites.sh [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+    "ram":"1Gi",
+    "defaultCommand": "/start.sh",
+    "instanceType": "m5.large",
+    "disk": 20,
+    "allowSensitive":false
+}
+EOF
+  set_tool_settings "$TEST_GROUP/$E2E_ENDPOINTS_TOOL" "$payload_e2e_endpoints"
+
+  local registry_id=$(get_registry_id "$TEST_GROUP/$E2E_ENDPOINTS_TOOL")
```
Comment (kamyshova) [inline e2e/gui/src/test/resources/prerequisites.sh]: Why do we need to reassign this value?

File: e2e/gui/src/test/resources/prerequisites.sh [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+    fi
+    echo "$get_run_response"
+    local run_id=$(echo "$get_run_response" | jq -r '.payload.id')
+    if [ "$run_id" ] && [ "$run_id" != "null" ]; then
+       echo -e "\n***Run pipeline-$run_id was successfully launched\n"
+       check_run $run_id
+    else
+       print_err "ERROR: Unable to get run id from the getting run response"
+       return 1
+    fi
```
Comment (kamyshova) [inline e2e/gui/src/test/resources/prerequisites.sh]: Unify the indent, please

File: e2e/gui/src/test/resources/utils.sh [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+    print_err "Error occurred while removing tool group with $tool name:"
+    echo "========"
+    echo "Response:"
+    echo "$get_tool_response"
+    echo "========"
+    return 1
+  else
+    echo "Tool group with $tool was deleted successfully"
+  fi
+}
```
Comment (kamyshova) [inline e2e/gui/src/test/resources/utils.sh]: Add an empty line to the end of the script, please

**Review by kamyshova — CHANGES_REQUESTED** [review]

File: e2e/gui/src/test/resources/prerequisites.sh (hunk 2 above)
Comment (maryvictol) [reply e2e/gui/src/test/resources/prerequisites.sh]: Extra reassigning has been removed

File: docs/testcases/UI/Test_Environment/TC-TEST-ENVIRONMENT-3.md (hunk 1 above)
Comment (maryvictol) [reply docs/testcases/UI/Test_Environment/TC-TEST-ENVIRONMENT-3.md]: Done

File: e2e/gui/src/test/resources/utils.sh (hunk 4 above)
Comment (maryvictol) [reply e2e/gui/src/test/resources/utils.sh]: Done

**Review by kamyshova — APPROVED** [review]

**Pull request merged**